tools/test-js-with-node

```diff
@@ -63,10 +67,63 @@ if not options.force:
         print('If you really know what you are doing, use --force to run anyway.')
         sys.exit(1)
 
+NODE_COVERAGE_PATH = 'var/node-coverage/coverage.json'
+INDEX_JS = 'frontend_tests/zjsunit/index.js'
+
 os.environ['NODE_PATH'] = 'static'
 os.environ['TZ'] = 'UTC'
 
-INDEX_JS = 'frontend_tests/zjsunit/index.js'
+if options.check_enforced:
+    coverage_json = None
+    print()
+    print("=============================================================================")
+    print("Checking for fully covered files that are not enforced yet...")
+    print("Running `test-js-with-node --coverage` first.")
+    print()
+
+    if args:
+        print('BAD ARGS! Coverage reports run against all files.')
+        sys.exit(1)
+
+    istanbul = os.path.join(ROOT_DIR, 'node_modules/.bin/istanbul')
+    command = [istanbul, 'cover', INDEX_JS, '--dir', 'var/node-coverage']
+
+    try:
+        ret = subprocess.check_call(command)
+    except OSError:
+        print('Bad command: %s' % (command,))
+        raise
+
+    try:
+        with open(NODE_COVERAGE_PATH, 'r') as f:
+            coverage_json = ujson.load(f)
+    except IOError:
+        print(NODE_COVERAGE_PATH + " doesn't exist.")
+        raise
```

Comment:
why not just move this block after the normal block that runs the tests?  Then you get to share all the code for running them...

Actually, I think if you combine this with the existing `--coverage` block, you can delete like 50 lines of duplicated code, since the loop is even the same.